src/debug-tools/initializer/debug-tools-init.tsx

```diff
@@ -105,13 +122,11 @@ const createStoreProxies = (storeUpdateMessageHub: StoreUpdateMessageHub) => {
     return [featureFlagStore, scopingStore, userConfigurationStore, permissionsStore];
 };
 
-const getStoreActionMessageCreator = (browserAdapter: BrowserAdapter, stores: BaseStore<any>[]) => {
-    const actionMessageDispatcher = new RemoteActionMessageDispatcher(
-        browserAdapter.sendMessageToFrames,
-        null,
-        createDefaultLogger(),
-    );
-
+const getStoreActionMessageCreator = (
+    browserAdapter: BrowserAdapter,
```

Comment:
nit: this impl doesn't actually use `browserAdapter` anymore. It might be cleaner to just inliine it, it's very simple and only used once.